src/main/java/nextstep/courses/domain/session/image/CoverImage.java

```diff
@@ -17,7 +19,19 @@ public static CoverImage of(String imageType, int imageFileSize, int width, int
         ImageFileSize fileSize = ImageFileSize.of(imageFileSize);
         ImageSize imageSize = ImageSize.of(width, height);
 
-        return new CoverImage(imageTypeEnum, fileSize, imageSize);
+        return new CoverImage(null, imageTypeEnum, fileSize, imageSize);
```

Comment:
다른 정적 팩토리 메서드를 호출하는 방식으로 변경해볼 수 있을 것 같습니다.
```suggestion
        return of(null, imageType, imageFileSize, width, height);
```